internal/config/config.go

```
@@ -20,7 +20,8 @@ type Config struct {
 		HomeserverName  string `yaml:"homeserver_name"`
 		AppServiceToken string `yaml:"as_token"`
 		HomeserverToken string `yaml:"hs_token"`
-		BotActorID      string `yaml:"bot_actor_id"` // Bot's Alkemio UUID, used as Matrix localpart
+		BotActorID      string `yaml:"bot_actor_id"`      // Bot's Alkemio UUID, used as Matrix localpart
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)